internal/payloads/account_payload.go

```diff
@@ -9,7 +9,14 @@ import (
 )
 
 type AccountPayload struct {
-	*models.Account
+	// Required auto-generated PK.
+	ID int64 `db:"id" json:"id"`
```

Comment:
This struct will never be stored in the db, so don't tag it please:

```suggestion
	ID int64 `json:"id"`
```

The same applies below.